rs/nns/governance/src/lib.rs

```diff
@@ -243,13 +244,24 @@ impl IcClock {
     }
 }
 
+fn time_u64_nanos() -> u64 {
+    if cfg!(target_arch = "wasm32") {
+        ic_cdk::api::time()
+    } else {
+        SystemTime::now()
+            .duration_since(SystemTime::UNIX_EPOCH)
+            .expect("Failed to get time since epoch")
+            .as_nanos()
+            .try_into()
```

Comment:
```suggestion
        u64::from(
            SystemTime::now()
                .duration_since(SystemTime::UNIX_EPOCH)
                .expect("Failed to get time since epoch")
                .as_nanos()
        )
```

Reply:
I don't believe this works, b/c Duration.as_nanos returns a u128